QUEST_JOBSTEP.tsv

```diff
@@ -109,42 +109,42 @@ QUEST_JOBSTEP_20150317_000108	You will find Merogs at the Fallen Worry Forest.{n
 QUEST_JOBSTEP_20150317_000109	The quality is fine enough. The Archer Master will be pleased. {nl}Here, I've prepared a copy, so take it. 
 QUEST_JOBSTEP_20150317_000110	Now there is one book left. {nl}Give this pouch to the Ranger Master and get the book. {nl}
 QUEST_JOBSTEP_20150317_000111	Did the Archer Master send you? {nl}Well, good work. I'll give you the book as promised. 
-QUEST_JOBSTEP_20150317_000112	Isn't the book ready yet? {nl}I want to see all of Lydia Schaffen's book ASAP...
-QUEST_JOBSTEP_20150317_000113	Finally it's all here. {nl}I'll teach you my skills as promised. 
-QUEST_JOBSTEP_20150317_000114	I have some books to collect. It's Lydia Schaffen's books. {nl}I'll teach you my skills if you collect the books. What do you say? 
-QUEST_JOBSTEP_20150317_000115	The first book was in the Miner's Village but it disappeared after the Vubbes attacked. {nl}I assume the book is in the Vubbe Outpost? 
+QUEST_JOBSTEP_20150317_000112	$Isn't the book ready yet? {nl}I want to see all of Lydia Schaffen's book as soon as possible...
+QUEST_JOBSTEP_20150317_000113	$Finally, it's all here. {nl}I'll teach you my skills as promised. 
+QUEST_JOBSTEP_20150317_000114	$I have some books to collect. They're Lydia Schaffen's books. {nl}I'll teach you my skills if you collect the books. What do you say? 
+QUEST_JOBSTEP_20150317_000115	$The first book was in the Miners' Village, but it disappeared after the Vubbes attacked. {nl}I assume the book is in the Vubbe Outpost? 
```

Comment:
Here, correct tense is used. I'm not sure the last '?' fits the context. They assume it's there, just like in other lines, so why is it a question? It would make sense if you could confirm it at that point, but otherwise?